gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa/factors-select-dialog/factors-select-dialog.component.ts

```diff
@@ -64,6 +64,10 @@ export class FactorsSelectDialogComponent implements OnInit {
     return this.factors?.length > 0;
   }
 
+  noneFactorSelected(): boolean {
```

Comment:
I suggest to add the map type 

41 line: 
  model: Map<string, boolean>;


46 line: 
   this.model = this.data.factors.reduce((model, factor) => {
      model[factor.id] = factor.selected;
      return model;
    }, new Map());

Reply:
Fixed.